src/Main.hs

```diff
@@ -85,6 +88,8 @@ data Opts = Opts
   { optsToken :: AuthToken
   , optsOffline :: Bool
   , optsIgnoreCase :: Bool
+  , optsSearchAll :: Bool
+  , optsShowDate :: Bool
   , optsBy :: Maybe String
   , optsOrganisation :: ParamName Organisation
   , optsFlow :: ParamName Flow
```

Comment:
I don't think it can, but feel free to prove me wrong. Sure the param can be Maybe, but you can't start giving the needle before filling out flow in the current structure.